rotkehlchen/tests/unit/decoders/test_zksync.py

```diff
@@ -104,3 +104,86 @@ def test_zksync_lite_deposit(database, ethereum_inquirer, ethereum_accounts):
         ),
     ]
     assert expected_events == events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+def test_zksync_lite_withdrawal(database, ethereum_inquirer, ethereum_accounts):
+    """Test a transaction with the Withdrawal event"""
```

Comment:
nitpick. If test/function is welll named, as is this case, there is no need for a docstring repeating the same info

```suggestion
```